app/src/main/java/com/sduduzog/slimlauncher/ui/main/HomeFragment.kt

```diff
@@ -188,6 +188,7 @@ class HomeFragment(private val viewModel: MainViewModel) : BaseFragment(), OnLau
 
     override fun onAppClicked(app: App) {
         launchApp(app.packageName, app.activityName, app.userSerial)
+        app_drawer_edit_text.clearFocus() // dismiss the on-screen keyboard
```

Comment:
```suggestion
        app_drawer_edit_text.clearFocus() // dismiss the on-screen keyboard
        app_drawer_edit_text.clearComposingText()
```

Seems like we should also clear out any text that was entered (so the user does not have to delete it for the next search).